java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualReaderImpl.java

```diff
@@ -182,28 +182,31 @@ void onRemoteReaderEvent(final ReaderEvent event) {
 
         logger.debug("{} EVENT {} ", this.getName(), event.getEventType());
 
-        if (thisReader.countObservers() > 0) {
-            /*
-             * thisReader.notifyObservers(event);
-             */
-
-
-            // launch event another thread to permit blocking method to be used in update methode
-            // (such as transmit)
-            executorService.execute(new Runnable() {
-                @Override
-                public void run() {
-                    thisReader.notifyObservers(event);
-                }
-            });
-
-
-        } else {
-            logger.debug(
-                    "An event was received but no observers are declared into VirtualReader : {} {}",
-                    thisReader.getName(), event.getEventType());
-        }
+        // TODO Check this!!!
+        if (thisReader instanceof VirtualObservableReader) {
```

Comment:
to remove the test (thisReader instanceof VirtualObservableReader): not necessarry if VirtualObservableReaderImpl extends VirtualReaderImpl